docs/reference/run.md

```diff
@@ -200,10 +200,29 @@ on the system.  For example, you could build a container with debugging tools
 like `strace` or `gdb`, but want to use these tools when debugging processes
 within the container.
 
-    $ docker run --pid=host rhel7 strace -p 1234
+### Example: run strace inside a container
 
-This command would allow you to use `strace` inside the container on pid 1234 on
-the host.
+Create this Dockerfile:
+
+```
+FROM alpine:3.2
+RUN apk add --update strace && rm -rf /var/cache/apk/*
+ENTRYPOINT ["strace"]
+```
+
+Build the Dockerfile and tag the image as `mystrace`:
+
+```bash
+$ docker build -t mystrace .
+```
+
+Use the following command to run `strace` inside the container on pid 1234 on
+the host:
+
+```
+$ docker run --pid=host --privileged mystrace -p 1234
+Process 1234 attached
```

Comment:
I have a question here,  Does pid mnt etc namespace share would need privileged flag ?  I thought it not necessary.

If pid not bind with privileged flag, I suggest give a example not need privileged flag.